src/app/components/feed/FeedDataPoints.js

```diff
@@ -0,0 +1,278 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, FormattedHTMLMessage } from 'react-intl';
+import CheckFeedDataPoints from '../../CheckFeedDataPoints';
+import FeedDataPointsSection from './FeedDataPointsSection';
+import Alert from '../cds/alerts-and-prompts/Alert';
+import { stringHelper } from '../../customHelpers';
+import styles from './SaveFeed.module.css';
+
+const FeedDataPoints = ({ readOnly, dataPoints, onChange }) => {
+  const toggleDataPoint = (enabled, dataPoint) => {
```

Comment:
It took me a minute to figure out what's happening here but I like what you're doing by essentially just removing the dataPoint from the array and then adding it back in if we are toggling it on.